packages/pn-pa-webapp/src/components/NewNotification/__test__/PreliminaryInformationsContent.test.tsx

```diff
@@ -0,0 +1,131 @@
+import { Formik, FormikProps } from 'formik';
+import { vi } from 'vitest';
+
+import {
+  fireEvent,
+  getById,
+  render,
+  screen,
+  testFormElements,
+} from '@pagopa-pn/pn-commons/src/test-utils';
+
+import { NewNotificationLangOther } from '../../../models/NewNotification';
+import { PreliminaryInformationsPayload } from '../../../redux/newNotification/types';
+import PreliminaryInformationsContent from '../PreliminaryInformationsContent';
+
+// mock imports
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (str: string) => str,
+    i18n: { language: 'it' },
+  }),
+}));
+
+describe('PreliminaryInformationsContent', () => {
+  const mockFormikProps = {
+    values: {
+      lang: 'it',
+      subject: '',
+      abstract: '',
+      additionalLang: '',
+      additionalSubject: '',
+      additionalAbstract: '',
+    },
+    touched: {},
+    errors: {},
+    handleChange: jest.fn(),
+  } as unknown as FormikProps<PreliminaryInformationsPayload>;
+
+  const mockLanguages = {
+    it: 'Italiano',
+    de: 'Tedesco',
+  };
+
+  const mockOnChangeTouched = jest.fn();
+  const mockSubjectHelperText = 'Helper text for subject';
+
+  it('renders the component', () => {
+    const result = render(
+      <Formik initialValues={mockFormikProps.values} onSubmit={() => {}}>
+        <PreliminaryInformationsContent
+          formik={mockFormikProps}
+          languages={mockLanguages}
+          onChangeTouched={mockOnChangeTouched}
+          subjectHelperText={mockSubjectHelperText}
```

Comment:
Why are we passing the subjectHelperText when we pass the formik instance to the component?

Reply:
I always need to show helper text when input is focused, not only when there’s an error

Reply:
and the formik istance is not enough?

Reply:
I don't have that string in formik

Reply:
Ok and we can't move that string inside the component?